### PR TITLE
new blog post by alsadi about skopeo+ostree+oci

### PR DESCRIPTION
**Pull request by muayyad-alsadi**

### CODE REVIEW

File: source/blog/2018-03-02-containers-w-skopeo-ostree-oci.md [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+You can list running Bubble wrapped containers using `bwrap-oci list`
+
+```
+$ bwrap-oci list
+NAME                          PID       STATUS    BUNDLE
+redis                         23369     running   /home/alsadi/ostree/cont1
+```
+
+Unfortunately there is no `bwrap-oci exec`, 
+
```
Comment (giuseppe) [inline source/blog/2018-03-02-containers-w-skopeo-ostree-oci.md]: can you add a section explaining that this is what "atomic install --system|--user" does behind the scenes and show the command for doing it?

Comment (muayyad-alsadi) [reply source/blog/2018-03-02-containers-w-skopeo-ostree-oci.md]: done

**Comment by muayyad-alsadi** [comment]
kindly review the article [here](https://github.com/muayyad-alsadi/atomic-site/blob/alsadi-blog-1/source/blog/2018-03-02-containers-w-skopeo-ostree-oci.md)

**Comment by rhatdan** [comment]
LGTM

**Comment by muayyad-alsadi** [comment]
> Fix typos and some wording.

thank you

File: source/blog/2018-03-02-containers-w-skopeo-ostree-oci.md [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+## Atomic Options
+
+Atomic Install has corresponding options to the choices we have demonistrated in this artcile like
```
Comment (miabbott) [inline source/blog/2018-03-02-containers-w-skopeo-ostree-oci.md]: s/artcile/article/

File: source/blog/2018-03-02-containers-w-skopeo-ostree-oci.md [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+redis                         23369     running   /home/alsadi/ostree/cont1
+```
+
+Unfortunately there is no `bwrap-oci exec`.
+
+## Atomic Options
+
+Atomic Install has corresponding options to the choices we have demonstrated in this article like:
+
+- `--storage=ostree|docker` weither to use `docker` or `ostree` to store the image
```
Comment (miabbott) [inline source/blog/2018-03-02-containers-w-skopeo-ostree-oci.md]: s/weither/either/

Comment (muayyad-alsadi) [reply source/blog/2018-03-02-containers-w-skopeo-ostree-oci.md]: fixed

File: source/blog/2018-03-02-containers-w-skopeo-ostree-oci.md (hunk 2 above)
Comment (muayyad-alsadi) [reply source/blog/2018-03-02-containers-w-skopeo-ostree-oci.md]: fixed by @sanjabonic

File: source/blog/2018-03-02-containers-w-skopeo-ostree-oci.md [hunk 4]
```diff
@@ -0,0 +1,310 @@
...
+ociimage/busybox_3Alatest
+```
+
+The Atomic command like tool is written in python, and it uses `libostree` via `gobject-introspection`, it looks like [this](https://github.com/projectatomic/atomic/blob/v1.22/Atomic/syscontainers.py#L26).
+
+```
+import gi
+gi.require_version('OSTree', '1.0')
+from gi.repository import OSTree
+```
```
Comment (miabbott) [inline source/blog/2018-03-02-containers-w-skopeo-ostree-oci.md]: Lines 118-124 seem out of place in the overall flow of the article.  Is there a reason to include this snippet of code while discussing using the `ostree` CLI?

Comment (muayyad-alsadi) [reply source/blog/2018-03-02-containers-w-skopeo-ostree-oci.md]: it says that atomic which is written python uses ostree via `gi` not using something like `subprocess`

**Comment by sanjabonic** [comment]
I'd say merge - if no other comments/edits come within 1.5 hours, I'll push it.

**Pull request merged**